tests/pytests/test_doctable.py

```diff
@@ -5,7 +5,7 @@
 # mainly this test adding and removing docs while the doc table size is 100
 # and make sure we are not crashing and not leaking memory (when runs with valgrind).
 def testDocTable():
-    env = Env(moduleArgs='MAXDOCTABLESIZE 100')
+    env = Env(moduleArgs='MAXDOCTABLESIZE 100 WORKER_THREADS 1 ENABLE_THREADS TRUE')
```

Comment:
Threads are enabled for just a handful of tests? Ideally shouldn't this be enabled in addition to the existing tests which do not use threads?

Reply:
isn't it redundant? These tests' main purpose is to check that everything still works as expected and that things that weren't in the scope of this PR are not affected
(in some of the tests we just initialize the thread pool without doing anything in the background, like testCursors)

Reply:
I meant that as long as we also support the current behavior, which is without threads, we should keep testing it. If we are adding new behavior with threads, we should test it in addition, not instead of.

@rafie @DvirDukhan We should discuss how to cover the matrix when more options are available - ideally we just multiply all tests by `SEARCH_THREADS` by `WORKER_THREADS` but we should probably find some heuristic.

Reply:
K
I'm removing this change until we have a test mechanism for multi-threading (somewhere in the following PRs)